### PR TITLE
Fixed issue #947 by editing type strings

### PR DESCRIPTION
**Pull request by tuethan1999**
### Pull Request Description
- Edited the type strings of the Unknown, EmailAddress, FilePath, IPAddress, SubRegionCode

### CODE REVIEW

**Comment by rwedge** [comment]
After this change, the `Variable` class is appearing in the dictionary returned`find_variable_types`.    Since the `Variable` class isn't a variable type we expect users to use as an actual variable, let's exclude it from the `find_variable_types` dictionary.  I think `ft.primitives.get_aggregation_primitives` and `ft.primitives.get_transform_primitives` do something to ensure `TransformPrimitive` and `AggregationPrimitive` don't get returned with their subclasses.

**Comment by rwedge** [comment]
https://github.com/FeatureLabs/featuretools/blob/e9f12ed71799ccad9e842cd7cf170d362407f056/featuretools/entityset/deserialize.py#L30

Since we've changed the type string for `Unknown`, we should update this line.

I also think this edge case -- being unable to find a variable that matches the type string in a variable json -- is untested.  We should add a test case.

**Comment by rwedge** [comment]
Let's also add a line to the breaking changes section of the changelog, detailing which Variables had their type strings changed.

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -26,6 +28,10 @@ Changelog
   no longer used to determine the time column. Now, both instance id columns and time columns in a cutoff time
   dataframe can be in any order as long as they are named properly.
 
+* The type_string attributes of all entity classes are now a snake case conversion of their class names. This
```
Comment (rwedge) [inline docs/source/changelog.rst:31]: * variable classes, not entity classes
* let's use code formatting (\`\`text\`\`) around type_string and the formal class names (Unknown, IPAddress, etc)
* Instead of talking about schema versions, in the last sentence let's say that old saved entitysets that used these variable may load incorrectly
